docs/modules/mosdepth.md

```diff
@@ -0,0 +1,34 @@
+---
+Name: mosdepth
+URL: https://github.com/brentp/mosdepth
+Description: >
+    Mosdepth performs fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing.
+    It can output:
```

Comment:
This description is too long sorry - it's used in the table on the MultiQC homepage, so should be max one short-ish line.

Reply:
Hah, got it! Done.